README_KR.md

```diff
@@ -0,0 +1,104 @@
+# euphony-listener
+​
+## 소개
+​
+Euphony-Listener는 Speaker로부터 음파를 통해 데이터를 받아 인식하는 기능을 합니다.</br>
+스피커가 데이터를 보낼 때 주파수가 발생하게 되는데, Listen 버튼을 누르면 Listener가 데이터를 인식하여 그 결과를 확인할 수 있습니다.</br>
+android에서만 동작을 합니다.</br>
+​
+## Euphony 라이브러리
+​
+해당 음파 데이터 통신 라이브러리, Euphony의 버전은 안드로이드를 위한 버전입니다.</br>
+Euphony는 마이크와 녹음기를 활용하여 다른 단말기(안드로이드, 웹)들과 통신하기 위해 디자인 된 무선 라이브러리를 제공합니다.</br>
+​
+공식 페이스북 페이지 : https://www.facebook.com/euphonyproject</br>
+공식 라이브러리 주소 : https://dev.jbear.co/euphony</br>
+​
+## 전제 조건
+​
+**build.gradle**
+```java
```

Comment:
```gradle
dependencies {
    implementation 'euphony.lib:euphony:0.7.1.6'
}
```

As you know, here is gradle script.
Please modify language name for code-highlight :)

Reply:
What I mean was like below
```suggestion
```gradle
```

When it comes to line no 19, the previous commit seems better.